source/Files.cpp

```diff
@@ -321,7 +320,7 @@ FILE *Files::Open(const filesystem::path &path, bool write)
 {
 #if defined _WIN32
 	FILE *file = nullptr;
-	_wfopen_s(&file, Utf8::ToUTF16(path.string()).c_str(), write ? L"w" : L"rb");
+	_wfopen_s(&file, path.string().c_str(), write ? L"w" : L"rb");
```

Comment:
No need for the `.string()`, just `path.c_str()` https://en.cppreference.com/w/cpp/filesystem/path/native